packages/shared/lib/core/wallet/utils/generateActivity/helper/updateFromPartialActivity.ts

```diff
@@ -0,0 +1,14 @@
+import { ActivityType } from '@core/wallet/enums'
+import { Activity } from '@core/wallet/types'
+
+export function updateFromPartialActivity(activity: Activity, partialData: Partial<Activity>): void {
```

Comment:
Is this an artefact from a merge I cant see it being used?

Reply:
yes